Tests/Fragments/FragmentFormFieldTestTrait.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/*
+ * This file is part of the Sonata project.
+ * 
+ * (c) 2018 - Sonata
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SonataHelpers\Tests\FragmentService;
+
+use Sonata\AdminBundle\Admin\AdminInterface;
+use Sonata\AdminBundle\Form\FormMapper;
+use Sonata\ArticleBundle\Entity\Fragment;
+use Sonata\ArticleBundle\FragmentService\FragmentServiceInterface;
+use Sonata\CoreBundle\Form\Type\ImmutableArrayType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * Trait FragmentFormFieldTestTrait
+ *
+ * @author Sylvain Rascar <sylvain.rascar@ekino.com>
+ */
+trait FragmentFormFieldTestTrait
```

Comment:
Same as previous class, this is not directly related to sonata, might have its place in a specific repo.

Reply:
It is related to SonataArticle

Reply:
Yes my bad, missed the fragment part :)